git-cinnabar

```diff
@@ -8,12 +8,19 @@ if test -z "$py"; then
   done
 fi
 if test -z "$py"; then
-  for py in python3 python python2.7 python2; do
-    command -v "$py" > /dev/null && break
+  for py in python3 python python2.7 python2 py.exe; do
```

Comment:
How about something like the following?
```
for py in python3 python python2.7 python2 "py -3" "py -2"; do
  $py -c "import sys" > /dev/null 2>&1 && break
  py=python3
done
```

Reply:
The issue there is that it means we have to treat `$py` as containing both the executable and any arguments. That's fine for the case where we auto detect Python. However, if the user sets GIT_CINNABAR_PYTHON to a full path (which might include spaces on Windows), that's going to break.

Reply:
That could be solved by using a different branch for GIT_CINNABAR_PYTHON